### PR TITLE
Update client.py

### PR DESCRIPTION
**Pull request by fabio-silva-movile**
New purpose, case insentive and unique transition parameter

### CODE REVIEW

**Comment by fabio-silva-movile** [comment]
I cant test because the local test are firing the following error:

tests/tests.py::PEP8-check(ignoring E501 E265 E127 E901 E128) FAILED

File: jira/client.py [hunk 1]
```diff
@@ -909,8 +909,26 @@ def transitions(self, issue, id=None, expand=None):
...
+        Get a transitionid available on the specified issue to the current user.
+        Look at https://developer.atlassian.com/static/rest/jira/6.1.html#d2e1074 for json reference
+        
+        :param issue: ID or key of the issue to get the transitions from
+        :param trans_name: iname of transition we are looking for
+        """
+        transitions_json = this.transitions(issue)
+        id = None
+
+        for transition in transtitions_json["transtions"]:
```
Comment (sargentstudley) [inline jira/client.py]: @fabio-silva-movile - FYI: you have a few syntax errors / mispellings of variable names in your request, such as here on this line.

Comment (fabio-silva-movile) [inline jira/client.py]: Ok I will fix this, many thanks

**Comment by ssbarnea** [comment]
Please update it and I will merge once is ok.

**Comment by fabio-silva-movile** [comment]
I have updated it on the same branch, this doesnt work? I need to generate a new pull request?

**Comment by ssbarnea** [comment]
No need as you probably observed. Now I am working to make the unittest run again and once they are done I will release a new version.

**Comment by fabio-silva-movile** [comment]
Okm thanks Sorin

**Pull request merged**